hw/ip/flash_ctrl/dv/flash_ctrl_base_sim_cfg.hjson

```diff
@@ -170,5 +170,12 @@
       name: smoke
       tests: ["flash_ctrl_smoke"]
     }
+    {
+      // For test clean up run subset of tests
+      name: sub
```

Comment:
nit: perhaps use a more meaningful name, like all_smoke or baisc_tests

BTW, you can also use regex to run these 2 tests in one shot - `-i *_smoke*`. `-i` supports regex.